src/main/java/com/exasol/projectkeeper/validators/changelog/ChangelogFile.java

```diff
@@ -0,0 +1,124 @@
+package com.exasol.projectkeeper.validators.changelog;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * This class represents a doc/changes/changes_x.x.x.md file.
+ */
+public class ChangelogFile {
```

Comment:
This name is confusing because we have `changelog` file. And it a different one...